ansible/roles/webapp/tasks/main.yml

```diff
@@ -42,6 +42,8 @@
     repo: "{{ app.webrepo }}"
```

Comment:
Where these values are pulled from?

Reply:
"the updates are rate limited to 4 times per hour" -- where do we define this? or this is default time?

Reply:
> Where these values are pulled from?

`app.webrepo` comes from `ansible/group_vars/appservers`

Reply:
> "the updates are rate limited to 4 times per hour" -- where do we define this? or this is default time?

This is defined in the system which is running these.